src/client/containers/SessionContainer.tsx

```diff
@@ -152,9 +152,8 @@ export const SessionContainer: React.SFC<SessionContainerProps> = ({
             ) {
               await setupSession(
                 {
+                  ...realApolloClient!,
                   client,
-                  subscriptionClient: realApolloClient!.subscriptionClient,
-                  httpLink: realApolloClient!.httpLink,
```

Comment:
Why this? Doesn't seem related to tracking, no?

Reply:
since I added a new field on ApolloClientUtils needed to add it here as well

Reply:
also, we're running client.mutate in some places, maybe it would be useful to also wrap that at some point